string mission_name
string waypoint_name
---
